samples/ask-akka-agent/src/main/java/akka/ask/agent/application/ConversationHistoryView.java

```diff
@@ -49,13 +49,13 @@ public Effect<Session> onEvent(SessionMemoryEntity.Event event) {
     }
 
     private Effect<Session> aiMessage(SessionMemoryEntity.Event.AiMessageAdded added) {
-      Message newMessage = new Message(added.message(), "ai", added.timestamp());
+      Message newMessage = new Message(added.message(), "ai", added.timestamp().toEpochMilli());
```

Comment:
not sure how the timestamp is used here, but maybe we could keep it as Instant ?

Reply:
Its used on the view and then send back to the UI.. I can change it but need to go a bit further and update the UI side to properly parse the date. Let me know if you think it's worth.